src/pages/dar_application/DataAccessRequestApplication.jsx

```diff
@@ -21,10 +21,7 @@ import { Storage } from '../../libs/storage';
 import { assign, cloneDeep, get, head, isEmpty, isNil, isString, keys, map } from 'lodash/fp';
 import './DataAccessRequestApplication.css';
 
-import {
-  validateDARFormData
-} from '../../utils/darFormUtils';
-import { isArray, set } from 'lodash';
+import {validateDARFormData} from '../../utils/darFormUtils';
```

Comment:
Imports can be shortened here and in other components as well
```suggestion
import {validateDARFormData} from 'src/libs/utils/darFormUtils';
```